src/sentry/static/sentry/app/utils/getStacktraceBody.jsx

```diff
@@ -0,0 +1,33 @@
+import rawStacktraceContent from 'app/components/events/interfaces/rawStacktraceContent';
+
+export default function getStacktraceBody(event) {
+  if (!event || !event.entries) {
+    return [];
+  }
+
+  // TODO(billyvg): This only accounts for the first exception, will need navigation to be able to
+  // diff multiple exceptions
+  //
+  // See: https://github.com/getsentry/sentry/issues/6055
+  const exc = event.entries.find(({type}) => type === 'exception');
+
+  if (!exc) {
+    // Look for a message if not an exception
+    const msg = event.entries.find(({type}) => type === 'message');
+    if (!msg) {
+      return [];
+    }
+    return msg.data && msg.data.formatted && [msg.data.formatted];
```

Comment:
@billyvg it seems like `formatted` is expected now instead of message - just wanted to point this change out